### PR TITLE
python updates

### PR DESCRIPTION
**Pull request by simonpintarelli**
- fix / cleanup of nlcg python
- add new pytest

### CODE REVIEW

File: src/unit_cell/unit_cell.hpp [hunk 1]
```diff
@@ -329,6 +331,17 @@ class Unit_cell
         return *atom_types_[id__];
     }
 
+    inline bool
+    has_augmented_atom() const
```
Comment (toxa81) [inline src/unit_cell/unit_cell.hpp]: Unit_cell::augment() provides same functionality

**Comment by toxa81** [comment]
@simonpintarelli please have a look at the comment about `has_augmented_atom()`. Otherwise it's good to go

**Comment by simonpintarelli** [comment]
@toxa81 which comment?

File: src/unit_cell/unit_cell.hpp (hunk 1 above)
Comment (toxa81) [reply src/unit_cell/unit_cell.hpp]: @simonpintarelli here

**Review by toxa81 — CHANGES_REQUESTED** [review]

File: src/unit_cell/unit_cell.hpp (hunk 1 above)
Comment (simonpintarelli) [reply src/unit_cell/unit_cell.hpp]: Thanks. I've removed my duplicate.

**Review by toxa81 — APPROVED** [review]
LGTM

**Pull request merged**